src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: auto;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $primary-color;
+    padding-top: 40px;
+    margin: 0;
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
+    margin: 0;
+    padding: 4px 0 16px;
+  }
+
+  &__reviews-block {
+    display: flex;
+    justify-content: space-between;
+    height: 16px;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    line-height: 14px;
+    color: $primary-color;
+    margin: 0;
+  }
+
+  &__value {
+    justify-content: space-between;
+    display: flex;
+    padding-bottom: 16px;
+  }
+
+  &__price {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+    margin: 0;
+  }
+
+  &__price-number {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $primary-color;
+    align-items: end;
+    margin: 0;
+  }
```

Use scss nesting everywhere
```suggestion
  &__price {
    font-size: 12px;
    line-height: 18px;
    color: $secondary-color;
    margin: 0;
    
    &-number {
      // write styles here
    }
  }
```